pkg/util/openshift.go

```diff
@@ -18,24 +18,33 @@ package util
 
 import (
 	"context"
+	"fmt"
 
 	configv1 "github.com/openshift/api/config/v1"
 	opv2 "github.com/operator-framework/api/pkg/operators/v2"
 	"github.com/operator-framework/operator-lib/conditions"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func DetermineOpenShiftVersion(ctx context.Context, client client.Client) (string, error) {
-	// Determine ocp version
-	clusterVersionList := configv1.ClusterVersionList{}
-	if err := client.List(ctx, &clusterVersionList); err != nil {
+func DetermineOpenShiftVersion(ctx context.Context, cl client.Client) (string, error) {
+	clusterVersion := &configv1.ClusterVersion{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "version",
+		},
+	}
```

Comment:
Type and Object Meta is embedded, unless you are defining more than the meta better separate the op:
```
clusterVersion := &configv1.ClusterVersion{}
clusterVersion.Name = "version"
```
this left aligns the assignment, which you can't go wrong w/.